README.md

```diff
@@ -1,153 +1,98 @@
+  integration_name          = "uptycs-int-123"
   organization_id           = "<GCP-ORGANIZATION-ID>"
 
-  # Set "integration_projects" to list of project IDs to integrate with uptycs
-  # Use provided python script to generate projects list with advanced filter options
-  integration_projects      = ""
+  # Select an existing project from your organization to host resources created by this configuration
+  host_project_id = "<Host-Project-ID>"
 
   # AWS account details
   # Copy Uptycs's AWS Account ID and Role from Uptycs' UI.
   # Uptycs' UI: "Cloud"->"GCP"->"Integrations"->"ORGANIZATION INTEGRATION"
   host_aws_account_id     = "<AWS account id>"
-  host_aws_instance_role  = "<AWS role>"
+  host_aws_instance_roles  = ["<Role1>", "<Role2>"]
 }
 
 output "host-project-id" {
   value = module.create-gcp-cred.host-project-id
 }
 
-output "regenerate-cred-config-command" {
-  value = module.create-gcp-cred.regenerate-cred-config-command
-}
-
-output "integration-projects-list" {
-  value = module.create-gcp-cred.integration-projects-list
+output "integration-name" {
+  value = module.create-gcp-cred.integration-name
 }
 
-output "integration-projects-list-command" {
-  value = module.create-gcp-cred.integration-projects-list-command
+output "credential-file-gen-command" {
+  value = module.create-gcp-cred.credential-file-gen-command
 }
 
 ```
-### Notes:-
-  * For more input parameters please follow below ##Inputs section and modify if required.
-  * By default, with an empty `integration_projects`, ALL active projects are integrated.
-  * It is recommended to pass comma-separated list of project ids so specific projects are integrated.
-  * Python script is available (check repo) generate project list with the help of advanced filters. 
-
-### 6.Execute Terraform script to get credentials.json and project-list.json
-```
-$ terraform init
-$ terraform plan  # Warning :- Please verify before applying.
-$ terraform apply
-# NOTE 1: Once terraform successfully applied, it will create "credentials.json" and "project-list.json" files
-# NOTE 2: retry 'apply' if it fails with following error
-#   - "Error creating WorkloadIdentityPool: googleapi: Error 403: Permission"
-```
-
-### Inputs
+### 2. Init, Plan and Apply
 
+#### Inputs
 | Name                      | Description                                                          | Type          | Default          |
 | ------------------------- | -------------------------------------------------------------------- | ------------- | ---------------- |
 | organization_id           | The GCP parent organizations id where resources will be created.     | `string`      | Required            |
 | integration_name          | Unique phrase. Used to name resources                                | `string`      | `"uptycs-int-1"`    |
-| service_account_name      | The service account name which will be created in host project.      | `string`      | `"sa-for-uptycs"`|
```

Comment:
It has been removed. We are using it. No info about host-project-id  in inputs

Reply:
added it back